src/events.rs

```diff
@@ -60,7 +68,6 @@ pub enum Event {
     /// The parameter is true if app was suspended, and false if it has been resumed.
     Suspended(bool),
```

Comment:
> the problem with Suspended and Awakened is that I need to change the backends' code if I remove them.

I guess it's probably OK to leave for a follow-up PR then. You could do it on another branch and wait for contributors from each backend to do PRs to that branch if you can't do each backend yourself.

> As for the keyboard events, the windows compositor usually sends the event only to the currently focused window, so sending it to a specific window is how it's designed.

Now that you mention it, this also seems to be the way macOS windows work too.